src/north/CsvToHttp/CsvToHttp.class.js

```diff
@@ -174,12 +267,47 @@ class CsvToHttp extends ApiHandler {
           object.error.push(`Header "${mapping.httpField}": ${response.error}`)
         }
         object.value[field] = response.value
+      } else if (mapping.csvField.match(REGEX_CONTAIN_VARIABLE_STRING)) {
+        // split the input into a array of string
+        // "test ${value}" => [ "test ", ${value}]
+        const csvFieldSplit = mapping.csvField.split(REGEX_SPLIT_TEMPLATE_STRING).filter(Boolean)
+        const field = mapping.httpField
+        csvFieldSplit.forEach((element) => {
+          if (element.match(REGEX_MATCH_VARIABLE_STRING)) {
```

Comment:
comment this (same comment than the one at the top of the file) to explain what you get here